client_lib/pulp/client/extensions/exceptions.py

```diff
@@ -126,8 +126,14 @@ def handle_bad_request(self, e):
                     'provided the following error message:')
             self.prompt.render_failure_message(msg)
 
-            self.prompt.render_failure_message('   %s' % e.error_message)
-            msg = _('More information may be found using the -v flag.')
+            msg = '   %s' % e.error_message
```

Comment:
I don't think so there would be an extra empty ``self.prompt.render_failure_message(msg)``